src/libraries/System.Text.Json/tests/Common/ReferenceHandlerTests/ReferenceHandlerTests.Serialize.cs

```diff
@@ -30,6 +30,7 @@ public class Employee
         }
 
         [Fact]
+        [ActiveIssue("Newtonsoft.Json baseline not supported in Native AOT", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
```

Comment:
Should we make this a ConditionalFact instead? This isn't really an "ActiveIssue". Newtonsoft.Json is not supported, intentionally. So it isn't really an "issue".

Reply:
Good question, I tried using it but I couldn't find a way to condition it on the negation of a boolean. Guess I need to add a `IsNotNativeAot` property?

Reply:
Ah, looks like somebody has already added that :-)